### ISSUE TITLE
Struggling to learn using Opt. Flow

### ISSUE THREAD

**Issue by mazatov** [deepmind/kinetics-i3d#122]
I'm training this model on my own dataset. I trained it successfully on the RGB stream of the data. Now, I"m trying got do that on the optical flow stream. However, the model is not learning anything at all. 

My optical flow output is scaled from [-1,1] and is of `float32` type. I'm detecting it using `method=cv2.cuda.FarnebackOpticalFlow_create( numLevels = 10, pyrScale = 0.5, winSize = 1, numIters = 20 )` for speed. By visualizing it, I can see that it is detecting something coherent. However, the model is not learning absolutely anything. Given that I'm using already pre-trained weights, my thought is that maybe the input is not what the model is expecting and hence can't learn anything because it stats on "the wrong foot". Could there be anything I'm missing regarding preprocessing steps or others?

Or is the model just very sensitive the to the flow method and it needs to have TVL1 as a flow method?

**Comment by leenas233** [comment]
hello ,I am a beginner. I wonder how to get the optical flow datasets. are there any websites to download the dataset or should I process the raw video to get the optical flow data? Thanks a lot!

**Comment by joaoluiscarreira** [comment]
Hey Mike,

it may be worth checking if the input range is the same as the model was
pretrained on. Maybe also play a bit with optimization parameters, maybe
the learning rate is too high ? I assume you mean that the training loss is
not going down ("the model is not learning absolutely anything").

Best,

Joao
